wordpress-mobile/WordPress-iOS
# Feature/7236 no results view dynamic type

Fixes part of #7236

This PR implements dynamic type in the (old) `WPNoResultsView`.
There is still the issue with the (extra extra) extra extra extra large font, but since this class will be replaced soon, I decided to not give it too much time.

![artboard](https://user-images.githubusercontent.com/9772967/37416415-7976432a-278c-11e8-9cb0-53b82d27d333.png)

To test:
- Get a `No result view`
  - The easiest is from Media -> Search (something that you don't have)
  - Other option is Site Pages (or Blog Posts) -> Scheduled
- Check it by changing the system font size.
